tests/test_ops.py

```diff
@@ -69,4 +69,4 @@ def test_kernel_generation(self, equation, expected):
 
         operator = OperatorOPS(eval(equation))
 
-        assert str(operator._ops_kernels[0]) == expected
+        assert str(operator.hcode) == expected
```

Comment:
where is the `hcode` property defined?

Reply:
Wasn't using it as a property, changed it. Thanks for poiting out, pushing the changes now